src/pages/dataset-details-page/index.tsx

```diff
@@ -183,8 +183,21 @@ const DatasetDetailsPage: FC<Props> = ({
     }
   }, [dataset?.id, isMounted]);
 
+  const dataServicesRelations = filterRelations(relations, Entity.DATA_SERVICE);
+  const datasetsRelations = filterRelations(relations, Entity.DATASET);
+  const publicServicesRelations = filterRelations(
```

Comment:
Brukes denne?

Reply:
Er kanskje bedre å bare bruke denne i stedet for `isDescribedAtServiceRelations`?

Reply:
Jeg ser det gjøres både kall til /search + searchType og /search/dataservices. Hva er forskjellene?